math/gnuplot/Portfile

```diff
@@ -129,7 +129,16 @@ variant old_bitmap_terminals description "Enable PBM (Portable Bit Map) and othe
     configure.args-replace      --without-bitmap-terminals --with-bitmap-terminals
 }
 
-default_variants                +aquaterm +luaterm +pangocairo +wxwidgets +x11
+default_variants                +luaterm +pangocairo +x11
+
+# aquaterm is currently not building on 10.4
+if {${os.platform} eq "darwin" && ${os.major} > 8} {
+    default_variants-append     +aquaterm
+}
+# wxWidgets 3.0 requires 10.5 or later (or non-Darwin)
```

Comment:
(In theory you could use wxWidgets 2.8 on 10.4, but I doubt that anyone really cares.)